torch/onnx/symbolic_helper.py

```diff
@@ -247,19 +249,65 @@ def _interpolate_size_to_scales(g, input, output_size, dim):
     output_size = _maybe_get_const(output_size, 'is')
     if _is_value(output_size):
         offset = 2
-        offsets = g.op("Constant", value_t=torch.ones(offset))
+        offsets = g.op("Constant", value_t=torch.ones(offset, dtype=torch.float32))
         dividend = g.op("Cast", output_size, to_i=cast_pytorch_to_onnx["Float"])
-        divisor = _slice_helper(g, g.op("Shape", input), axes=[0], ends=[dim], starts=[offset])
+        divisor = _slice_helper(g, g.op("Shape", input), axes=[0], ends=[9223372036854775807], starts=[offset])
         divisor = g.op("Cast", divisor, to_i=cast_pytorch_to_onnx["Float"])
         scale_dims = g.op("Div", dividend, divisor)
         scales = g.op("Concat", offsets, scale_dims, axis_i=0)
     else:
         scales_constant = [1. if i < 2 else
                            float(output_size[-(dim - i)]) / float(input.type().sizes()[-(dim - i)])
                            for i in range(0, dim)]
-        scales = g.op("Constant", value_t=torch.tensor(scales_constant))
+        scales = g.op("Constant", value_t=torch.tensor(scales_constant, dtype=torch.float32))
     return scales
 
+def _interpolate_get_scales(g, scale_factor, dim):
```

Comment:
make _interpolate_get_scales and _interpolate_get_scales_and_mode_from_args consistent?

_interpolate_get_scales + _interpolate_get_scales_and_mode or 
_interpolate_get_scales_from_args + _interpolate_get_scales_and_mode_from_args